server/src/graphql/scannerTypes.js

```diff
@@ -134,6 +134,7 @@ module.exports = gql`
     display_pokemon_id: Int
     ditto_form: Int
     weight: Float
+    height: Float
```

Comment:
height is Float, size Int..?!

Reply:
Maintains backwards compatibility for now, since `size` for everyone not using the latest rdm is still going to be a float ;)